### ISSUE TITLE
MIB lack of whitespace cause untrapped parsing error, valueType not set

### ISSUE THREAD

**Issue by derrell** [markabrahams/node-net-snmp#174]
TL;DR: `SYNTAX INTEGER(0..128)` fails to parse the type because there is no whitespace after `INTEGER`

The following definition from IPV6-MIB fails to parse the SYNTAX because there is no space between `INTEGER` and `(0..128)`. The result is that `valueType` ends up as undefined rather than `2` (`ObjectType.Integer`) resulting in a GET returning null instead of the integer value set in the table column.

```
 ipv6AddrPfxLength OBJECT-TYPE
    SYNTAX      INTEGER(0..128)
    UNITS       "bits"
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION
      "The length of the prefix (in bits) associated with
      the IPv6 address of this entry."
    ::= { ipv6AddrEntry 2 }
```

**Comment by markabrahams** [comment]
Hi @derrell - long time haha!  Not sure they have valid SMIv2 there, but I'll give them the benefit of the doubt! :-)  I'll have a look at accounting for this in the parser.

**Comment by markabrahams** [comment]
Fixed now on master, and fix published in version 3.5.1 of the npm.

**Comment by derrell** [comment]
Thanks @markabrahams. Yeah, I don't know if it's valid or not, but it's in use in the wild so supporting it is a good thing.
